Atomic/scan.py

```diff
@@ -123,6 +125,14 @@ def get_additional_args():
 
         self.results_dir = os.path.join(self.results, self.scanner, self.cur_time)
 
+        # Get remediation script
+        if self.args.remediate:
+            for i in self.scanners:
+                self.remediation_script = i.get("remediation_script", None)
```

Comment:
shouldn't this break once it finds it? Or shouldn't it append to list of remediation scrpits? This code seems suspect.

Reply:
agree, nice comment